_posts/2018-08-23-childcare-update.md

```diff
@@ -0,0 +1,18 @@
+---
+author: David Fischer
+category: General
+date: 2018-08-23 06:00:00
+layout: post
+image: /static/img/blog/childcare.png
+title: "Childcare updates"
+---
+
+Back in June, we [announced our childcare survey](/news/childcare-survey/) to ask about the needs of our attendees when it comes to childcare at DjangoCon US. Childcare at conferences and DjangoCon US in particular is an important factor that helps folks attend who otherwise may not be able to come. We gathered your feedback, spoke with local childcare providers, and we have some updates about childcare at DjangoCon US.
+
+The San Diego Marriott pointed us to [Destination Sitters](https://www.destinationsitters.com/), a service where certified and vetted sitters come directly to your hotel room and bring a bag of toys for your little ones to play with. There's no need to arrange a pick-up and drop-off at a faraway daycare.
+
+* **Booking:** Destination Sitters recommends you book a sitter 2-3 weeks in advance although they may be able to accommodate shorter notice. To book a sitter, you can [book online](https://www.destinationsitters.com/book-a-hotel-babysitter/) or call (888) SIT-KIDZ.
```

Comment:
Maybe wrap the telephone number in a `<a href="tel:+123456789">Phone</a>` thing so people can click to call?

Reply:
Especially since it's a vanity alphabetical number